homeassistant/components/sensor/co2mini.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for the CO2 and temperature sensor of CO2meter's CO2Mini USB indoor air
+quality monitor and compatible devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.co2mini/
+"""
+
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_DEVICE, CONF_FRIENDLY_NAME, STATE_UNKNOWN
+from homeassistant.core import callback
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['https://github.com/jannau/CO2Meter/archive/fea592cfd2b54124e4afdc20320f5c929e1c07d6.zip#CO2Meter==2.0']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_DEVICE): cv.string,
+    vol.Optional(CONF_FRIENDLY_NAME): cv.string,
+})
+
+DATA_CO2MINI_DEV = 'co2mini_dev'
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up CO2 mini Sensor."""
+    import CO2Meter
+
+    device = config.get(CONF_DEVICE)
+    name = config.get(CONF_FRIENDLY_NAME, device.rsplit('/', 1)[-1])
+
+    sensors = {}
+    sensors[CO2Meter.CO2Meter_CO2] = CO2MiniSensor(name + " CO2", "ppm")
+    sensors[CO2Meter.CO2Meter_TEMP] = CO2MiniSensor(name + " Temperature", "°C")
```

Comment:
This [specs](https://www.co2meter.com/collections/fixed-wall-mount/products/co2mini-co2-indoor-air-quality-monitor)? I guess it could depend on the sensor configuration/firmware. My device returns the temperature as kelvin and the zero point in converted in the library.